pkg/github/github.go

```diff
@@ -792,18 +810,25 @@ func (g *GitHub) ListReleaseAssets(
 
 // TagExists returns true is a specified tag exists in the repo
 func (g *GitHub) TagExists(owner, repo, tag string) (exists bool, err error) {
-	tags, _, err := g.Client().ListTags(
-		context.Background(), owner, repo, &github.ListOptions{PerPage: 100},
-	)
-	if err != nil {
-		return exists, errors.Wrap(err, "listing repository tags")
-	}
+	options := &github.ListOptions{PerPage: 100}
```

Comment:
Is `100` the max value allowed here? Should we allow consumers of this package to limit the number of results returned if they don't need all pages?

Reply:
100 is the maximum per page, yes.
Of the three functions I'm patching, two return lists: `ListAssets` and `ListBranches`. The number of assets is always small, I don't think you would need to limit the returns.

In the case of branches, perhaps you would need to limit the returns if you had hundreds and hundreds of branches. Would it make sense?

Reply:
I think we can have a default value if the `PerPage` is not set by the client as we did in another pkg (https://github.com/cpanato/release/blob/master/pkg/vulndash/adapter/adapter.go#L88-L90)

Reply:
+1 for a default value and configuration options.